app/static/src/app/directives/tooltip-controlled.ts

```diff
@@ -0,0 +1,47 @@
+import { DirectiveBinding } from "vue";
+import { Tooltip } from "bootstrap";
+
+export interface ToolTipContent {
```

Comment:
```suggestion
export interface ToolTipSettings {
```
..since it doesn't just define the content..

Reply:
done! yh agree with this one!